src/antares/exceptions/exceptions.py

```diff
@@ -54,6 +54,15 @@ def __init__(self, area_name: str, message: str) -> None:
         super().__init__(self.message)
 
 
+class InvalidNameError(Exception):
```

Comment:
Name can contain these types of characters, it's the ids that cannot. What you should do is use transform_name_to_id(name) where you want to have an id but you shouldn't use this

So you'll have to adapt the rest of your code :/

Reply:
I've duplicated the functionality of AntaresWeb like asked, if that is no longer the case I wouldn't have made it this way.

![image](https://github.com/user-attachments/assets/e63b79b3-a2cb-4280-b7bd-541ee1ff3480)

I realize you can "force it" by using the API, but if for example you use a name with comma you can no longer correctly interact with your study using AntaresWeb (questionmark seems to work).

![image](https://github.com/user-attachments/assets/7691bc8c-6761-4ed5-bff5-2601532236c6)

Which leads to:

![image](https://github.com/user-attachments/assets/52b84cbd-10a1-4a60-9368-f60640fd4163)

Reply:
Indeed it's a bug inside AntaresWeb. But we shouldn't care about that inside antares-craft

Reply:
Removed.